src/test/java/LogInOutTest.java

```diff
@@ -7,25 +7,26 @@
 import pageclasses.MailPage;
 import pageclasses.WebDriverSingleton;
 
+import java.io.IOException;
+
 public class LogInOutTest {
     private final String USERNAME = "seleniumtrainingcs";
-    private final String PASSWORD = "Selenium23&*!#";
+    private final String PASSWORD = "Selenium23&*!#&";
     private LandingPage landingPage;
     private MailPage mailPage;
 
     @BeforeEach
     public void setup() throws InterruptedException {
-        // Common steps for all tests to avoid code duplication
         landingPage = new LandingPage();
         landingPage.openLandingPage();
         LoginPage loginPage = landingPage.clickSignIn();
         mailPage = loginPage.login(USERNAME, PASSWORD);
     }
 
-    // Both tests in same class due to similar steps and feature under test
     @Test
-    public void loginTest() {
+    public void loginTest() throws IOException {
         Assertions.assertEquals(USERNAME, mailPage.getUserAccountLabel());
+        mailPage.takeScreenshot();
```

Comment:
put it before assertion

Reply:
if assertion fail than next line will not run